src/library/simplified/extension/checkin-date.js

```diff
@@ -163,6 +163,11 @@ else if (utag.isItinPage()){
     if (b["entity.tripDetails.hotelOffer.isoFormatCheckInDate"] != undefined) {
         b['checkInDate'] = b["entity.tripDetails.hotelOffer.isoFormatCheckInDate"];
     }
+
+    //utc
+    if (b["entity.tripDetails.utcTripStartDate"] != undefined){
+        b['checkInDateUtc'] = b["entity.tripDetails.utcTripStartDate"];
```

Comment:
Can we call it check(In/Out)TimestampUtc instead of Date since it's technically seconds since epoch?